pkgs/top-level/all-packages.nix

```diff
@@ -23506,6 +23506,8 @@ with pkgs;
 
   libopenglrecorder = callPackage ../development/libraries/libopenglrecorder { };
 
+  libopenraw = callPackage ../development/libraries/libopenraw { };
```

Comment:
In addition to the other comments, can you use the new by-name structure here https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#readme ?